src/Text/index.jsx

```diff
@@ -26,9 +27,10 @@ const Text = ( {
             noWrap,
             role,
             size,
-            style
+            textAlign,
+            variant
         } ) }
-        style = { { letterSpacing: spacing, color: `#${textColor}` } }
+        style = { { letterSpacing, color: `#${textColor}` } }
```

Comment:
I think we should remove the `#` here. This way we can allow for colors specified in other formats (e.g. `rgb( x, y, z )`). So this can be simply:

```style = { { letterSpacing, color } }```